packages/react-scripts/config/webpack.config.js

```diff
@@ -38,6 +38,8 @@ const getCacheIdentifier = require('react-dev-utils/getCacheIdentifier');
 // @remove-on-eject-end
 const postcssNormalize = require('postcss-normalize');
 
+const commit = require('child_process').execSync('git rev-parse --verify HEAD').toString().trim();
```

Comment:
We should pass this in as a build variable, I think, rather than doing magic with git command lines... that way you don't have to have the git executables or the .git directory to do a build, which we wouldn't typically have if we were building in a docker container.